vnext/ReactUWP/Views/KeyboardEventHandler.cpp

```diff
@@ -0,0 +1,402 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "Views/KeyboardEventHandler.h"
+#include "Utils/Helpers.h"
+#include "Utils/PropertyHandlerUtils.h"
+#include <winrt/Windows.UI.Core.h>
+
+using namespace std::placeholders;
+
+static constexpr auto ALT_KEY = "altKey";
```

Comment:
Do these need to be aligned to the KeyboardEvent members? Seems like they intend to but it's not called out.

Reply:
nope. It's the string which will exchange with js. KeyboardEvent can be mapped to anything.